tests/integration/llm/prepare.py

```diff
@@ -594,7 +594,12 @@
         "engine": "MPI",
         "option.model_id": "s3://djl-llm/flan-t5-xxl-trtllm-compiled/v0.8.0/",
```

Comment:
0.9.0?

Reply:
Good catch. Updated and uploaded s3 s3://djl-llm/flan-t5-xxl-trtllm-compiled/v0.9.0/